es-ds-docs/pages/atoms/layout.vue

```diff
@@ -1,5 +1,328 @@
 <template>
     <div>
-        atoms - layout
+        <h1>
+            Layout
+        </h1>
+        <p>
+            Extended from 
+            <NuxtLink to="https://bootstrap-vue.org/docs/components/layout" target="_blank">
+                bootstrap-vue layout
```

Comment:
Nit: this is how it was before, but can we use the proper capitalization of "Bootstrap Vue" for the library name?